src/rules-parser.pegjs

```diff
@@ -140,13 +111,27 @@ PathExpression "path" =  parts:("/" part:PathKey? { return part; })+ {
     return part;
   });
   if (hasError) {
-    error((parts[parts.length - 1] === '' ? "Paths may not end in a slash (/) character"
+    error((parts[parts.length - 1] === ''
+           ? "Paths may not end in a slash (/) character"
            : "Paths may not contain an empty part") + ": /" + parts.join('/'));
   }
-  return parts;
+  return new ast.PathTemplate(parts);
 }
 
-PathKey = chars: [^ /;]+ { return chars.join(''); }
+PathKey = CaptureKey / LiteralPathKey
+
+CaptureKey = "{" _ id:Identifier _ ("=" _ "*" _ )? "}" {
```

Comment:
What is the subset of patterns we intend to support on CaptureKeys? 
I'm curious about patterns like the following: `{var = **}`, `{var = */*}`, `{var = */**}`